add scripts for ubuntu. (#129)
OpenVSLAM-Community/openvslam: scripts/ubuntu/Readme.md

## Download sample data to run tests
Before starting a docker, you need to download sample data.
Please run below:
```
./download_sampledata.sh
```

## Build OpenVSLAM
Now buid and start docker container as below.
```
./build_openvslam_docker.sh
```
### note
- If you don't have a GPU, edit the last line of "build_openvslam_docker.sh" as deleting "--gpu all".

## Run samples
If you finish the all above procedure, you will see the terminal inside a docker as "build_openvslam_docker.sh". Please input below commands in the terminal.
#### run tracking and mapping
```
./run_video_slam -v /vocab/orb_vocab.fbow -m /dataset/aist_living_lab_1/video.mp4 -c ../example/aist/equirectangular.yaml --frame-skip 3 --no-sleep --map-db map.msg
```
After you see the window is stopped, press "Terminate" button in the left pane.
You will get "map.msg" in the current directory.
You can use it for the below tracking demo.
#### run localization
```
./run_video_localization -v /vocab/orb_vocab.fbow -m /dataset/aist_living_lab_2/video.mp4 -c ../example/aist/equirectangular.yaml --frame-skip 3 --no-sleep --map-db map.msg
```

## Run OpenVSLAM on your own video

> If you want to run OpenVSLAM for your own mp4 video, put the panoramic transformed mp4 video in /dataset, and copy example/aist/equirectangular.yaml to example/yourown.yaml and edit the image size.

Reviewer: This simple configuration change steps are only supported for Equirectangular setup. Additional steps should be required for perspective, fisheye, stereo, and RGBD. Please consider describing it in a way that makes it clear.

Author: OK, I added some comment for other camera types.

> # This is an easy scripts to run samples from scratch for Ubuntu users.

Reviewer: Please change the file name to README.md.

Author: I'm sorry. Done.

> # Steps
> ## Clone the OpenVSLAM
> Probablly you already run git clone, but I redisplay the procedure to show you complete steps.

Reviewer: Please add a indentation.
https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines

```suggestion
# Steps

## Clone the OpenVSLAM

Probablly you already run git clone, but I redisplay the procedure to show you complete steps.
```

Please correct other similar sentences as well.

> ```

Reviewer: ```suggestion

```shell
```

Please correct other similar sentences as well.

## Run samples
If you finish all the above procedure, you will see the terminal inside a docker as "root@*****:/openvslam/build#". Please input below commands in the terminal.
#### run tracking and mapping
```
./run_video_slam -v /vocab/orb_vocab.fbow -m /dataset/aist_living_lab_1/video.mp4 -c ../example/aist/equirectangular.yaml --frame-skip 3 --no-sleep --map-db map.msg
```
After you see the window is stopped, press "Terminate" button in the left pane.
You will get "map.msg" in the current directory.
You can use it for the below tracking demo.
#### run localization
```
./run_video_localization -v /vocab/orb_vocab.fbow -m /dataset/aist_living_lab_2/video.mp4 -c ../example/aist/equirectangular.yaml --frame-skip 3 --no-sleep --map-db map.msg
```

## Run OpenVSLAM on your own video
If you want to run OpenVSLAM for your own mp4 video, for example equirectangular camera, put the panoramic transformed mp4 video in /dataset, and copy example/aist/equirectangular.yaml to example/yourown.yaml and edit the image size.
Then run ./scripts/ubuntu/build_openvslam_docker.sh and run the above "run tracking and mapping" command but changing to your mp4 and yaml file.

> For other camera type like perspective, fisheye, stereo, and RGBD, it will be almost the same procedure but copy other yaml file instead of example/aist/equirectangular.yaml. You can find several samples in examle.

Reviewer: ```suggestion
For other camera type like perspective, fisheye, stereo, and RGBD, it will be almost the same procedure but copy other yaml file instead of example/aist/equirectangular.yaml. You can find several examples in openvslam/examples.
```
